governance/polls/Ratification Poll for Modify Core Unit Budget, RISK-001 (MIP40c3-SP56) - February 14, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Modify Core Unit Budget, RISK-001 (MIP40c3-SP56) - February 14, 2022
+summary: MIP40c3-SP56 modifies the DAI budget for Core Unit RISK-001: Risk, replacing MIP40c3-SP13 
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp56-modify-core-unit-budget-risk-001/12587
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-02-14T16:00:00
+end_date: 2022-02-28T16:00:00
+---
+# Ratification Poll for Modify Core Unit Budget, RISK-001 (MIP40c3-SP56) - February 14, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, February 14 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP56 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP56 modifies the DAI budget for Core Unit RISK-001: Risk. The proposed monthly budget for the Risk Core Unit is 230,000 DAI per month. The document contains the following: (i) why we are proposing a modified budget, (ii) past work review, (iii) a breakdown of the proposed budget, and (iv) comparing the previous budget to the proposed budget modifications.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/2a7ad1fd7dec681a4b93c4677b3abc36cc98d2f2/MIP40/MIP40c3-Subproposals/MIP40c3-SP56.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP56)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp56-modify-core-unit-budget-risk-001/12587)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, February 24. 
```

Comment:
The date here is wrong, the vote doesn't finish until the 28th, so it would be confirmed on Thursday, March 3rd